content/post/career-ladders-part-3.md

```diff
@@ -0,0 +1,57 @@
+---
+author: Caitlin Moorman
+title: "How to Use Your Career Ladder"
```

Comment:
It might be worth calling out explicitly that someone can be doing a great job at their current level, but not be on the path to the next level, and it's important to be clear in your communication when that's the case. Again, I say this having learned that lesson in an extremely hard way. Saying "you're doing great" all year long lead to a really tough conversation around review time when we were mis-aligned about progression toward promotion.

Reply:
100% agree. Might also be worth talking about navigating the tricky and really unfair situation at high growth companies where the bar may move over time and what would have earned that person a promotion a year ago isn't going to earn a promotion today. But this could almost be another blog post.